src/components/Inventory/withdraw/flow/WithdrawalBatchesCellRenderer.tsx

```diff
@@ -41,12 +41,13 @@ export default function WithdrawalBatchesCellRenderer(props: RendererProps<Table
         <Box display='flex' alignItems={row.error[id] ? 'start' : 'center'}>
           <Textfield
             id={id}
-            type='text'
+            type='number'
```

Comment:
our number input is currently broken
you can't enter 0

Reply:
I tested entering 0 and it worked as expected.  In which case it doesn't work?

Reply:
interesting, it never worked for me.. this line https://github.com/terraware/web-components/blob/main/src/components/Textfield/Textfield.tsx#L124 changes the 0 to '' (i may have added it because we did not want to display a default 0, this change is still wrong anyways).
will try in the vercel link.. in any case, if it works good

Reply:
it works in vercel.. i should probably switch to this on the reassignments page

Reply:
Should we remove the `|| ''` from the value? Because if we do that, we maybe start showing zeros in some places we don't wan't

Reply:
yes we should probably remove it and fix the places where we were initializing to 0